packages/react-server-gulp-module-tagger/index.js

```diff
@@ -22,12 +22,13 @@ module.exports = function(config) {
 	config.basePath = module.filename.replace(THIS_MODULE,'');
 	return forEach(function(stream, file){
 		return stream.pipe(replace(REPLACE_TOKEN, (match, optString) => {
+			const opts = optString ? {label: optString.replace(/.+label:\s?"(\w+)".+/, "$1")} : undefined;
```

Comment:
Sounds good to me. Let's file the multi-line options issue? It's probably on the challenging side, but I think building on top of this PR would make it super narrowly-scoped, so there might be someone out there who wants to tackle it.